lib/test_bot.rb

```diff
@@ -110,22 +109,18 @@ def run!(args)
       end
 
       if tap
+        tap_github = " #{ENV["GITHUB_REPOSITORY"]}"
         tap_revision = Utils.safe_popen_read(
           GIT, "-C", tap.path.to_s,
           "log", "-1", "--format=%h (%s)"
         ).strip
-        puts Formatter.headline("Testing #{tap.full_name} #{tap_revision}:", color: :cyan)
+        puts Formatter.headline("Testing #{tap.full_name}#{tap_github} #{tap_revision}:", color: :cyan)
       end
 
       ENV["HOMEBREW_GIT_NAME"] = args.git_name || "BrewTestBot"
       ENV["HOMEBREW_GIT_EMAIL"] = args.git_email ||
                                   "1589480+BrewTestBot@users.noreply.github.com"
 
-      puts Formatter.headline("Tap configuration:", color: :cyan)
-      puts "GITHUB_REPOSITORY: #{ENV["GITHUB_REPOSITORY"]}"
-      puts "Core Tap: #{CoreTap.instance}"
-      puts "Found tap: #{tap}"
```

Comment:
@iMichka FYI I've rolled this into the existing information:
- CoreTap was already output unless it was the tap being tested (in which case it says "Testing" rather than "Using")
- I've added `GITHUB_REPOSITORY` output if it's set
- Whether the tap was found is indicated by "Testing" in the tap output